lib/libc/sys/select.c

```diff
@@ -5,29 +5,22 @@
 #include <stdlib.h>
 #include <unistd.h>
 #include <errno.h>
+#include <signal.h>
 
-int select(int nfds, fd_set *restrict readfds, fd_set *restrict writefds,
-           fd_set *restrict exceptfds, struct timeval *restrict timeout) {
+int pselect(int nfds, fd_set *readfds, fd_set *writefds,
+            fd_set *exceptfds, const struct timespec *timeout,
+            const sigset_t *sigmask) {
```

Comment:
nit: non consistent `const` and `restrict` modifiers for pointers - see `select`s arguments.

Reply:
Added missing `restrict` modifiers for `pselect` pointer arguments - https://github.com/cahirwpz/mimiker/pull/1419/commits/8e37006c10e6c623b5b485d018ee0f6208e83cb4